src/models/Spec.d.ts

```diff
@@ -11,6 +11,7 @@ declare interface RetryOptions {
   /** delay between retries - defaults to 3 */
   delay?: number;
   strategy?: string | RetryHandlerFunction;
+  status?: string | Array;
```

Comment:
type should be number.

```suggestion
  status?: number | number[];
```